test/appengine/test_gae_connectionpool.py

```diff
@@ -0,0 +1,193 @@
+import unittest
+import logging
+import sys
+
+from nose.plugins.skip import SkipTest
+
+try:
+    from google.appengine.api import urlfetch
+except ImportError:
+    raise SkipTest("App Engine SDK not available.")
```

Comment:
As the gae connectionpool is now in `contrib` you can move the tests to the contrib tests.
Then they won't be picked up automatically anymore.